suite-native/transactions/src/components/TransactionDetail/TransactionDetailData.tsx

```diff
@@ -60,13 +54,15 @@ export const TransactionDetailData = ({ transaction }: TransactionDetailDataProp
                     <TransactionDetailRow title="Fee">
                         <Box alignItems="flex-end">
                             <Text color="gray1000">
-                                {CryptoAmountFormatter.format(fee, {
```

Comment:
Prefer component version of formatters.

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/1cc34593cd42d928be9c517ad6ad4da0f00dc9a3